src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -282,13 +294,9 @@ to the `<a>` tag. On Android, links with `target="_blank"` open in a
 A few other notes on `scope`:
 
 * If you don't include a `scope` in your manifest, then the default implied
-  `scope` is the directory that your web app manifest is served from.
-* The `scope` attribute can be a relative path (`../`), or any higher level
-  path (`/`) which would allow for an increase in coverage of navigations
-  in your web app.
+  `scope` is the start URL but with its filename, query, and fragment removed.
+* If `scope` attribute is a relative path, the manifest URL is used as a base URL.
 * The `start_url` must be in the scope.
-* The `start_url` is relative to the path defined in the `scope` attribute.
-* A `start_url` starting with `/` will always be the root of the origin.
```

Comment:
I believe this statement is irrelevant for `scope` so I removed it

Reply:
it is not irrelevant, it explains that when creating a `start_url` `/` doesn't mean 'from the scope' it means 'from the origin'